network-api/networkapi/templates/pages/base.html

```diff
@@ -1,11 +1,22 @@
+{% load wagtailmetadata_tags i18n localization %}
+
+{% get_current_language as lang_code %}
+
 <!DOCTYPE html>
-<html>{% load settings_value %}
+<html lang="{{ lang_code }}">{% load settings_value %}
   <head>
     <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1">{% block socialMetadata %}{% endblock %}
+    <title>{% block pageTitle %}Mozilla Foundation - {% if page.specifics.seo_title %}{{ page.specifics.seo_title }}{% else %}{{ page.title }}{% endif %}{% endblock %}</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    {% block social_metadata %}
+      <meta property='og:type' content='website'>
+      <meta property='og:locale' content='{{ lang_code|to_opengraph_locale }}'>
+      <meta property='wagtail:language' content='{{ lang_code }}'>
+      {% meta_tags %}
```

Comment:
What's `meta_tags` here?

Reply:
the tag used by the wagtail-metadata app to inject all the additional metadata fields from the `Promote` tab.

Reply:
actually, let me move that into its own block.

Reply:
Ah ok, that makes sense. Thanks!